### PR TITLE
[BIOMAGE-1747] Expired token

### PR DESCRIPTION
**Pull request by kafkasl**
# Description
The issue was caused by the `sendNotification` hook. This hook uses the `jwtToken` to get the current user information to know who to send the email notifications too (and also to add the user name to our slack notifications). Whenever, the token was expired, the hookRunner failed, crashing the qc|gem2s response handlers. 

Changes
* `sendNotification` now ignores if the `jwtToken` is expired because it only cares about the user info
* `hookRunner.run()` runs inside a try/catch to prevent any hook problem from propagating and interrupting the original calling function (hooks are by definition side-effects, so any error should not stop the main program).

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-1747

#### Link to staging deployment URL (or set N/A)
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [x] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [x] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [x] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [x] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [x] Started end-to-end tests on the latest commit.

### Documentation updates
- [x] Relevant Github READMEs updated **or** no GitHub README updates required.
- [x] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [x] Staging environment is unstaged before merging.
- [x] Photo of a cute animal attached to this PR.

image

### CODE REVIEW

File: src/api/route-services/pipeline-response.js [hunk 1]
```diff
@@ -128,13 +130,20 @@ class PipelineService {
...
+      console.log('getS3Output');
       output = await this.getS3Output(message);
+      console.log('updatePlotData');
       await this.updatePlotData(taskName, experimentId, output);
+      console.log('updateProcessingConfig');
       await this.updateProcessingConfig(taskName, experimentId, output, sampleUuid);
     }
+
+    console.log('and the rest is not silence');
+
```
Comment (ivababukova) [inline src/api/route-services/pipeline-response.js]: don't forget to remove the console logs

File: src/utils/hooks/send-notification.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// const jwt = require('jsonwebtoken');
```
Comment (ivababukova) [inline src/utils/hooks/send-notification.js]: remove the commented out line

**Review by ivababukova — APPROVED** [review]
remove the console logs and the commented out line and it's good to merge. Are these storks (the cute animals photo)?

**Comment by kafkasl** [comment]
> remove the console logs and the commented out line and it's good to merge. Are these storks (the cute animals photo)?

I assume they are haha japanese ones maybe

**Pull request merged**